e2e/rbd.go

```diff
@@ -3092,7 +3094,7 @@ var _ = Describe("RBD", func() {
 						framework.Failf("failed to check data pool for image: %v", err)
 					}
 
-					err = deleteJournalInfoInPool(f, pvc, "replicapool")
+					err = deleteJournalInfoInPool(f, pvc, rbdTopologyPool)
```

Comment:
`rbdTopologyPool` is not the `replicapool` either you are doing a bug fix or changing the behaviour needs to be cleared.

Reply:
I believe changing the behaviour, as Neils mentioned now the journal pool will be created on the topology pools